bedrock/settings/static_media.py

```diff
@@ -1301,6 +1316,28 @@
         ),
         'output_filename': 'js/firefox_firstrun-bundle.js',
     },
+    'firefox_firstrun_quantum': {
+        'source_filenames': (
+            'js/base/mozilla-fxa-iframe.js',
```

Comment:
I think you need to add `'js/base/uitour-lib.js',` here to invoke UITour

Reply:
yes, that explains it, thank you